pkg/parse/parse.go

```diff
@@ -518,7 +518,7 @@ func DefaultPlatform() string {
 // Platform separates the platform string into os, arch and variant,
 // accepting any of $arch, $os/$arch, or $os/$arch/$variant.
 func Platform(platform string) (os, arch, variant string, err error) {
-	if platform == "local" {
+	if platform == "local" || platform == "" || platform == "/" {
```

Comment:
It might be nice to have a debug statement in here just to let people know the error triggered and the value found in the platform variable.

Reply:
For these cases it will always return `default` without any errors but for any error case scenario it retruns error with the value used in `--platform` ( Line: 531 )